src/test/java/MongoDbTest.java

```diff
@@ -0,0 +1,179 @@
+import com.mongodb.MongoClient;
+import com.mongodb.MongoCredential;
+import com.mongodb.client.FindIterable;
+import com.mongodb.client.MongoCollection;
+import com.mongodb.client.MongoDatabase;
+import com.mongodb.client.model.Filters;
+import com.mongodb.client.model.Indexes;
+import com.mongodb.client.model.Projections;
+import org.bson.Document;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.*;
+
+import static com.mongodb.client.model.Filters.*;
+import static com.mongodb.client.model.Updates.*;
+
+public class MongoDbTest {
+    private MongoClient mongo;
+    private MongoCredential credential;
+    private MongoDatabase database;
+    private MongoCollection<Document> taskCollection;
+
+    @Before
+    public void init() {
+        mongo = new MongoClient("localhost", 27017);
+        credential = MongoCredential.createCredential("user", "database",
+                "password".toCharArray());
+        database = mongo.getDatabase("database");
+
+        taskCollection = database.getCollection("task");
+        taskCollection.drop();
+        taskCollection = database.getCollection("task");
+
+        insertData();
+    }
+
+    @Test
+    public void displayAllTasks() {
+        FindIterable<Document> iterDoc = taskCollection.find();
+        int i = 1;
+        Iterator it = iterDoc.iterator();
+        while (it.hasNext()) {
+            System.out.println(it.next());
```

Comment:
it's not a test if you haven't `assertions` there :-)

Reply:
it could print some output but how we could be sure that it's correct?)

Reply:
Added assertions.